heron/common/src/cpp/basics/spconsts.h

```diff
@@ -38,4 +40,20 @@ constexpr auto constTcpKeepAliveSecs = 300;          // 5 min
 constexpr auto constTcpKeepAliveProbeInterval = 10;  // 10 seconds
 constexpr auto constTcpKeepAliveProbes = 5;
 
+constexpr sp_int64 operator"" _KB(unsigned long long bytes) { // NOLINT
+    return (sp_int64)bytes * 1024;
+}
+
+constexpr sp_int64 operator"" _MB(unsigned long long bytes) { // NOLINT
```

Comment:
If I understand correctly, using `amount` for argument name instead of `bytes` makes better sense? (`100_MB` means _one hundreds_ megabytes. )